app/decorators/application_choice_export_decorator.rb

```diff
@@ -1,10 +1,18 @@
 class ApplicationChoiceExportDecorator < SimpleDelegator
+  RELEVANT_SUBJECTS = [
+    ApplicationQualification::MATHS,
+    ApplicationQualification::ENGLISH,
+    ApplicationQualification::SCIENCE,
+    ApplicationQualification::SCIENCE_SINGLE_AWARD,
+    ApplicationQualification::SCIENCE_DOUBLE_AWARD,
+    ApplicationQualification::SCIENCE_TRIPLE_AWARD,
+  ].freeze
+
   def gcse_qualifications_summary
-    gcses = %i[maths_gcse english_gcse science_gcse]
-    summary_string = gcses
-      .map { |gcse| application_form.send(gcse) }
+    summary_string = application_form
+      .application_qualifications
+      .select { |qualification| qualification.level == 'gcse' && RELEVANT_SUBJECTS.include?(qualification.subject) }
```

Comment:
The only change here would be the order of the GSCE qualifications we return in that field. Before it was ensured that the order was math, english then science (if available) now it could be anything. Should we retain the previous order, or actually the order doesn't matter at all?

Reply:
:+1: I did wonder about this, we don't appear to test for order but I think for consistency we can ensure that order is returned.

Reply:
@cwrw Addressed here https://github.com/DFE-Digital/apply-for-teacher-training/pull/5438/commits/dca2b50620ccf2733a192213748b057621b6b9d1